pkg/server/server.go

```diff
@@ -425,8 +425,13 @@ func (s *ProxyServer) serveRecvFrontend(stream client.ProxyService_ProxyServer,
 				klog.ErrorS(err, "Failed to get a backend", "serverID", s.serverID)
```

Comment:
Should probably add 
  "dialID", random 
to this error message.

Reply:
Good idea, done.